tools/gromacs/sim.xml

```diff
@@ -230,6 +231,7 @@
             <expand macro="md_inputs"/>
         </section>
 
+        <param name="use_gpu" value="false" type="boolean" hidden="true"/>
```

Comment:
We do not need this, we can do this purely on the Galaxy side. https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/sample/workflow_resource_params_conf.xml.sample

So just use GPU_AVAILABLE and make sure the tests are passing here. The other stuff we need to configure in Galaxy with @gmauro